OpenMEEG/src/assemble.cpp

```diff
@@ -268,25 +268,51 @@ int main(int argc, char** argv)
     * (i.e. the potential and the normal current on all interfaces)
     * |----> v (potential at the ECoG electrodes)
     **********************************************************************************************/
-    else if ( option(argc, argv, {"-Head2ECoGMat", "-H2ECogM", "-H2ECOGM", "-h2ecogm"},
-                     {"geometry file", "conductivity file", "ECoG electrodes positions file",
-                      "the name of the interface for EcoG", "output file"}) ) {
+    else if (argc==6 && option(argc, argv, {"-Head2ECoGMat", "-H2ECogM", "-H2ECOGM", "-h2ecogm"},
+                                           {"geometry file", "conductivity file", "ECoG electrodes positions file", "output file"})) {
 
         // Loading surfaces from geometry file.
+
         Geometry geo;
         geo.read(argv[2], argv[3], OLD_ORDERING);
 
-        // Find the mesh of the ECoG electrodes
-        const Interface &i = geo.interface(argv[5]);
+        // Reading the file containing the positions of the EEG patches
 
-        // read the file containing the positions of the EEG patches
         Sensors electrodes(argv[4]);
 
-        // Assembling Matrix from discretization :
+        // Assemble matrix from discretization:
+        // Head2ECoG is the linear application which maps x |----> v
+
+        std::cerr << "Warning: we assume that ECoG electrodes are placed on the inner interface." << std::endl
```

Comment:
```
(WARNING) New version of H2ECOGM option requires you to specify the name of the interface on which ECoG positions are positioned. Consider specifying an interface as a name
 right after the electrode position file. Without this information, we will consider that the ECoG electrodes are placed on the inner interface.
```
besides I read when running this PR with the demo ECoG model:

Detected interface: 2 3

I don't know what this means.

Reply:
It was a debug message I used to debug the example you gave me. It was not giving the same result. The interface 2 is the one detected as the inner one and "3" is the one that was in the script. Removed with the new commit.